api/PerformanceElementTiming.json

```diff
@@ -0,0 +1,532 @@
+{
+  "api": {
+    "PerformanceElementTiming": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceElementTiming",
+        "support": {
+          "chrome": {
+            "version_added": "77"
+          },
+          "chrome_android": {
+            "version_added": "77"
+          },
+          "edge": {
+            "version_added": "79"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "64"
+          },
+          "opera_android": {
+            "version_added": "55"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": "12.0"
+          },
+          "webview_android": {
+            "version_added": "77"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
```

Comment:
https://wicg.github.io/element-timing/ has "unofficial draft" written so large it's hard to read the spec. Whether WICG should get the `standards_track` status is the subject of https://github.com/mdn/browser-compat-data/issues/6103, but in this case I think the spec itself claims to be so unofficial that we should probably set it to false.

Reply:
I've gone ahead and updated the statuses in https://github.com/mdn/browser-compat-data/pull/7436/commits/2ed81ae2d0ad9fd3a993f7d1e2264b4d857c6ca8.